DDD.Functions.Extensions/TitoWebhookConfig.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Description;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace DDD.Functions.Extensions
+{
+    public class TitoWebhookConfig : Attribute
+    {
+        [AppSetting(Default = "TitoWebhookSecret")]
+        public string Secret { get; set; }
+
+        [AppSetting(Default = "TitoWebhookConnectionString")]
+        public string ConnectionString { get; set; }
+
+        [AppSetting(Default = "TitoWebhookDeDupeTable")]
+        public string DeDupeTable { get; set; }
+
+        [AppSetting(Default = "TitoWebhookOrderNotificationQueue")]
+        public string OrderNotificationQueue { get; set; }
+
+        [AppSetting(Default = "TitoWebhookTicketNotificationQueue")]
+        public string TicketNotificationQueue { get; set; }
+
+        [AppSetting(Default = "TitoApiToken")]
```

Comment:
We aren't using this yet, but I'd like to use it to get the tickets remaining count for each ticket type...

Reply:
That can be in a separate PR though...